index.js

```diff
@@ -49,7 +49,7 @@ module.exports = robot => {
         context.github.issues.getComments(githubHelper.parseCommentURL(issue.comments_url)).then(resp => {
           return freeze.getLastFreeze(resp.data);
         }).then(lastFreezeComment => {
```

Comment:
so this use case should not happen. if you got this far, that means you are in a thread that is frozen, so you should have a last frozen comment. The problem is upstream from this.